src/shogun/features/SNPFeatures.cpp

```diff
@@ -351,7 +351,7 @@ std::shared_ptr<Features> SNPFeatures::duplicate() const
 SGMatrix<float64_t> SNPFeatures::get_histogram(bool normalize)
 {
 	int32_t nsym=3;
-	float64_t* h= SG_CALLOC(float64_t, size_t(nsym)*string_length/2);
+	SGMatrix<float64_t> h(nsym, string_length/2,false);
```

Comment:
```suggestion
	SGMatrix<float64_t> h(nsym, string_length/2);
```

Reply:
nvm must of copied that one by mistake sorry :D